packages/containerd/containerd-config-toml_k8s

```diff
@@ -29,3 +29,11 @@ SystemdCgroup = true
 [plugins."io.containerd.grpc.v1.cri".cni]
 bin_dir = "/opt/cni/bin"
 conf_dir = "/etc/cni/net.d"
+
+{{#if settings.image-registry.mirrors}}
```

Comment:
nit: tildes at the start of the conditional lines to prevent extra blanks?

Reply:
With tildes at the start of `#if` will remove all white space before the handlebars expression causing the two setting blocks to merge like this:
```
[plugins."io.containerd.grpc.v1.cri".cni]
bin_dir = "/opt/cni/bin"
conf_dir = "/etc/cni/net.d"
[plugins."io.containerd.grpc.v1.cri".registry.mirrors]
    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."..."]
        endpoint = [.....]
    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."..."]
        endpoint = [.....]
```

Which deviates with how the rest of the config file looks. It doesn't change anything functionally though